### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by BenM4k**
In this request, we implemented all the codes for the popup window. I did the HTML and CSS part and my partner worked on the JavaScript

### CODE REVIEW

File: sass/works.scss [hunk 1]
```diff
@@ -245,6 +245,134 @@
...
+
+  &.active {
+    .overlay {
+      display: block;
+    }
+
+    .popup {
+      transition: all 0.3ms ease-in-out;
+      transform: translate(-50%, -70%) scale(1);
+    }
```
Comment (AbelG101) [inline sass/works.scss:373]: - [ ] Great job with the popup it's looking on point 🎯 but there seems to be a slight issue as the popup is not appearing in front of the screen but it appears either below or above the current screen so please fix that as it creates a frustrating experience for the user. 😟
- Here's how it currently looks:
       firefox_KYb4SwAP03

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @BenM4k & @Paul-tes 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- You're doing a really great job building your portfolio everything is looking perfect but you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -52,7 +52,8 @@ <h2>I'm a Software Developer</h2>
             </div>
         </section>
 
-        <section id="works" class="work_section flex_column">
+        <section id="works" class="flex_column work_section">
+            <div class="overlay"></div>
             <div class="work_head">
                 <h2 class="title">My Recent Works</h2>
                 <img src="assets/Vector.png" alt="vector" class="work_head_img">
```
Comment (NahnahAJ) [inline index.html:59]: - [ ] Good job so far @BenM4k and @Paul-tes. You did a good job with the popup functionality. However, As per the project requirements and as the previous reviewer mentioned, the projects section needs to be created dynamically. :+1: I have attached a screenshot below.
image

- [ ] This means the projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for the projects list should be removed, and instead, it should be added via JavaScript. :wink: 
- You can check an example of this student's project: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95

**Review by NahnahAJ — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @BenM4k and @Paul-tes , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
- Popup works as expected :heavy_check_mark:
- No linter errors :heavy_check_mark:
- Professional Readme :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @BenM4k, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**